src/server/worldserver/worldserver.conf.dist

```diff
@@ -3257,3 +3257,89 @@ LFG.Location.All = 0
 
 #
 ###################################################################################################
+
+###################################################################################################
+# MISC ANTI-CHEAT SETTINGS
+#    AntiCheats.FlyHack
+#        Description: Enable AntiCheat.FlyHack
+#        Default:     1 - (Enabled)
+#                     0 - (Disabled)
+
+AntiCheats.FlyHack.Enabled = 1
+
+#    AntiCheats.FlyHack
+#        Description: Enable AntiCheat.SpeedHack
+#        Default:     1 - (Enabled)
+#                     0 - (Disabled)
+
+AntiCheats.SpeedHack.Enabled = 1
+
+#    AntiCheats.DoubleJump.Enabled
+#        Description: Enable AntiCheat.DoubleJump Kick when player sent opcode jump when has flag falling (client can't do this)
+#        Default:     1 - (Enabled)
+#                     0 - (Disabled)
+
+AntiCheats.DoubleJump.Enabled = 1
+
+#    AntiCheats.FakeJumper.Enabled
+#        Description: Enable AntiCheat.FakeJumper : when player sent a move opcode (with new Z higher) when has flag falling (client can't sent any new packets when falling)
+#		      It's normal for Gagarin hacks and with Air Mode + "space" click = "falling" mode (player can't change orientation of flying, but it move up)
+#        Default:     1 - (Enabled)
+#                     0 - (Disabled)
+
+AntiCheats.FakeJumper.Enabled = 1
+
+#    AntiCheats.FakeFlyingmode.Enabled
+#        Description: Enable AntiCheat.FakeFlyingmode : when player sent a move opcode with added fly flag but on server this flag doesn't restricted for player.
+#		      It's normal for Hitchhiker's Hack with Air Mode
+#        Default:     1 - (Enabled)
+#                     0 - (Disabled)
+
+AntiCheats.FakeFlyingmode.Enabled = 1
+
+#    AntiCheats.IgnoreControlMovement.Enabled
+#        Description: Enable AntiCheat.IgnoreControlMovement : when player in root state - client can't send a new movement packets.
+#        Default:     0 - (Disabled)
+#                     1 - (Enabled)
+
+AntiCheats.IgnoreControlMovement.Enabled = 0
```

Comment:
is there any specific reason to set this as disabled by default (while all others are enabled) ?

Reply:
nope, just old texts. I forgot to update config texts like it updated on TC PR.